about.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/remixicon@2.5.0/fonts/remixicon.css" rel="stylesheet">
+    <link rel="stylesheet" href="abtt.css">
+    <title>Bole Festival 2023 About page</title>
+</head>
+
+<body>
+    <header>
+        <section class="header" id="header">
+            <nav class="nav-container1" id="nav-container1">
+                <div class="nav-menu1" id="nav-menu1">
+                    <ul class="nav-list1" id="">
+                        <li class="nav-item1"><a href="#">
+                                <i class="ri-facebook-fill"></i>
+                            </a></li>
+                        <li class="nav-item1"><a href="#">
+                                <i class="ri-twitter-fill"></i>
+                            </a></li>
+                        <li class="nav-item1"><a href="#">Register</a></li>
+                        <li class="nav-item1"><a href="#">Connect</a></li>
+                        <li class="nav-item1"><a href="#">Donate</a></li>
+                    </ul>
+                </div>
+            </nav>
+            <nav class="nav-container2 flex-nav" id="nav-container2">
+                <div class="nav-toggle" id="nav-toggle">
+                    <i class="ri-menu-fill"></i>
+                </div>
+                <ul class="nav-list2" id="nav-list2">
+                    <i class="ri-close-fill"></i>
+                    <li class="clicklogo"><a href="#"><img src="./img/bolefest.png" alt="logo"></a></li>
+                    <li class="click"><a href="./index.html">Home</a></li>
+                    <li class="click"><a href="#">Sponsors</a></li>
+                    <li class="click"><a href="#">News</a></li>
+                    <li class="clicks"><a href="#">FF Campaign</a></li>
+                </ul>
+            </nav>
+        </section>
+        <div class="sectiontwo">
+            <section2 class="homepage">
+                <h3>
+                    Back Bigger Better ...
+                </h3>
+                <h1>Bole Community Festival 2023!!! <br></h1>
+                <p>
+                    What happens at Bole Festival?
+                    Bole festival is a place where the southern food lovers converge to eat “Bole” the beloved and
+                    most popular street food in the South. We also create a safe and fun space for food lovers to
+                    meet food vendors. We always provide an assortment of fun games and competitions.
```

Comment:
- [ ]  You about top section should be centered column that includes a page title, a text block with the page description, and a line with a contact information or any other prompt under it, is it is provided from behance design
Your design 

![image](https://user-images.githubusercontent.com/43877689/227662507-a56598c6-2c80-46b8-934d-46beb874b432.png)

Behance design 
![image](https://user-images.githubusercontent.com/43877689/227662760-7c4ea804-da2c-41c0-a4c5-e66c2c9b57b9.png)